src/api/spec/jobs/update_released_binaries_job_spec.rb

```diff
@@ -10,11 +10,12 @@
     }
   end
 
-  let(:user) { create(:confirmed_user, :with_home, login: 'foo') }
-  let!(:repository) { create(:repository, project: user.home_project) }
+  let(:project) { create(:project, name: 'foo') }
+  let(:repository) { create(:repository, project: project) }
+  let(:event) { Event::Packtrack.create(project: project.name, repo: repository.name, payload: '12345') }
 
   describe '.update_binary_releases' do
```

Comment:
```suggestion
  describe '#perform' do
```
Merging `update_binary_releases` into `perform` also makes the `describe` section change.